Plugins/FormatSwift/Plugin.swift

```diff
@@ -16,10 +16,27 @@ struct AirbnbSwiftFormatPlugin: CommandPlugin {
 
     var argumentExtractor = ArgumentExtractor(arguments)
 
-    let inputPaths = try inputPaths(
-      context: context,
-      // Consumers can exclude additional subdirectories with an `--exclude` flag
-      excluding: argumentExtractor.extractOption(named: "exclude"))
+    // When ran from Xcode, the plugin command is invoked with `--target` arguments,
+    // specifying the targets selected in the plugin dialog.
+    let inputTargets = argumentExtractor.extractOption(named: "target")
```

Comment:
Are targets a list of relative paths?

Reply:
I think they're the names of targets as defined in the `Package.swift`, not paths or direcories